### PR TITLE
Add info to SwiftSignal docs

### PR DESCRIPTION
**Pull request by pratikmallya**

### CODE REVIEW

**Comment by vikomall** [comment]
@pratikmallya  you need to rebase this PR.

File: swift-signal-handle.rst [hunk 1]
```diff
@@ -41,7 +41,8 @@ Add a SwiftSignalHandle resource
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 SwiftSignalHandle is a resource to create temporary URL to receive
-notification/signals.
+notification/signals. Note that the temporary URL is created in Rackspace
```
Comment (rs-randallburt) [inline swift-signal-handle.rst]: Should be "Note that the temporary URL is created using Rackspace Cloud Files."

File: swift-signal-handle.rst [hunk 2]
```diff
@@ -69,11 +70,15 @@ temporary URL created by the above SwiftSignalHandle resource.
...
                 sleep 300
 
-                # Assuming long running operation completed successfully, notify success signal
+                # Assuming long running operation completed successfully,
+                # notify success signal
                 wc_notify --data-binary '{"status": "SUCCESS"}'
 
               params:
-                wc_notify: { get_attr: ['signal_handle', 'curl_cli'] 
+                # 'curl-cli' option will replace 'wc_notify' in the bash script
```
Comment (rs-randallburt) [inline swift-signal-handle.rst]: Could be clearer. Maybe something like:

  # Replace all occurances of "wc_notify" in the script with an appropriate curl 
  # PUT request using the "curl_cli" attribute of the SwiftSignalHandle

**Comment by rs-randallburt** [comment]
:-1:  minor nits wrt clarity

**Comment by rs-randallburt** [comment]
@pratikmallya can you either update or abandon this PR?

**Comment by rs-randallburt** [comment]
Sorry, didn't check the changes.

**Pull request merged**